### PR TITLE
Transforming "GraphQL create API" UI with react hooks

### PR DESCRIPTION
**Pull request by CrowleyRajapakse**
## Issue
product-apim: https://github.com/wso2/product-apim/issues/5458

## Methodology
use react hooks instead of using classes.

## UI Screenshots
- File Upload UI
Screen Shot 2019-09-03 at 5 30 38 PM

- After File uploaded
Screen Shot 2019-09-03 at 5 31 02 PM

- Create API step
Screen Shot 2019-09-03 at 5 31 53 PM

- Redirecting to created API overview page
Screen Shot 2019-09-03 at 5 32 12 PM

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Create/GraphQL/ApiCreateGraphQL.jsx [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+                                <Link to='/apis/'>
+                                    <Button variant='outlined'>
+                                        <FormattedMessage
+                                            id='Apis.Create.GraphQL.ApiCreateGraphQL.cancel'
+                                            defaultMessage='Cancel'
+                                        />
+                                    </Button>
+                                </Link>
+                            )}
+                            {wizardStep === 1 && <Button onClick={() => setWizardStep(step => step - 1)}>Back</Button>}
```
Comment (ruks) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Create/GraphQL/ApiCreateGraphQL.jsx]: 'Back' is a button text? if so don't we need to localize it?

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Create/GraphQL/ApiCreateGraphQL.jsx [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+                        </Grid>
+                        <Grid item>
+                            {wizardStep === 0 && (
+                                <Button
+                                    onClick={() => setWizardStep(step => step + 1)}
+                                    variant='contained'
+                                    color='primary'
+                                    disabled={!apiInputs.isFormValid}
+                                >
+                                    Next
```
Comment (ruks) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Create/GraphQL/ApiCreateGraphQL.jsx]: seems 'Next' need to be localized

Comment (CrowleyRajapakse) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Create/GraphQL/ApiCreateGraphQL.jsx:274]: resolved by #8330

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Create/GraphQL/ApiCreateGraphQL.jsx (hunk 1 above)
Comment (CrowleyRajapakse) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Create/GraphQL/ApiCreateGraphQL.jsx:264]: resolved by #8330

**Pull request merged**